ci-operator/jobs/infra-image-mirroring-multi-arch.yaml

```diff
@@ -0,0 +1,43 @@
+# DO NOT EDIT. This file is auto-generated.
+periodics:
+- agent: kubernetes
+  cluster: app.ci
+  cron: '@daily'
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+    workdir: true
+  labels:
+    ci.openshift.io/area: supplemental-ci-images-arm64
+    ci.openshift.io/role: infra
+  name: periodic-image-mirroring-supplemental-ci-images-arm64
+  spec:
+    containers:
+    - args:
+      - --skip-missing=true
+      - --filter-by-os=linux/arm64
```

Comment:
name multi-arch but only arm64 in the args

Reply:
We currently only have arm64 support. Once we add the new clusters, there will be more jobs in this file for different architectures.